io.catenax.planned_production_output/1.0.0/PlannedProductionOutput.ttl

```diff
@@ -0,0 +1,186 @@
+    samm:characteristic :QuantityCharacteristic.
+:productionSiteBpns a samm:Property;
+    samm:preferredName "BPNS of Supplier's Production Site"@en;
+    samm:description "One planned and alllocated production output has always one site. The site is identified by the BPNS."@en;
+    samm:characteristic :BPNSTrait;
+    samm:exampleValue "BPNS0123456789ZZ";
+    samm:see <https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_PDF_Maerz/5_BPDM/CX_-_0010_BUSINESS_PARTNER_NUMBER_PlatformCapabilityBPDM_v_1.0.1.pdf>.
+:estimatedTimeOfCompletion a samm:Property;
+    samm:preferredName "Estimated Time of Completion"@en;
+    samm:description "Date and time on which the planned and allocated production output is estimated to be completed. Date and time are set according to a timezone."@en;
+    samm:characteristic :DateTimeStamp;
+    samm:exampleValue "2023-04-01T14:23:00+01:00"^^xsd:date.
+:OrderIdCharacteristic a samm:Characteristic;
+    samm:preferredName "Order ID Characteristic"@en;
+    samm:description "This is a proprietary order ID, generated by either the supplier side or customer side."@en;
+    samm:dataType xsd:string.
+:OrderPositionIdCharacteristic a samm:Characteristic;
+    samm:preferredName "Position ID within Order"@en;
+    samm:description "The position ID is unique for an order and used within business context to refer to a position."@en;
+    samm:dataType xsd:string.
+:QuantityCharacteristic a samm-c:Quantifiable;
+    samm:preferredName "Quantity Characteristic"@en;
+    samm:description "Describes the quantity and the unit of measure with which the planned and allocated production output is measured."@en;
+    samm:dataType :Quantity.
+:Quantity a samm:Entity;
+    samm:preferredName "Quantity"@en;
+    samm:description "Comprises the number of objects and the unit of measurement."@en;
+    samm:properties (:quantityNumber :unitOfMeasure).
+:unitOfMeasure a samm:Property;
+    samm:preferredName "Unit of Measure"@en;
+    samm:description "Unit of measurement for demand quantities."@en;
+    samm:characteristic :UnitOfMeasure;
+    samm:exampleValue "KGM".
+:UnitOfMeasure a samm-c:Enumeration;
+    samm:preferredName "Unit of Measure"@en;
+    samm:description "The Enumeration lists a selection of allowed units of measure referenced by their official code according to Codes for Units of Measure Used in International Trade"@en;
+    samm:dataType xsd:string;
+    samm:see <https://tfig.unece.org/contents/recommendation-20.htm>;
+    samm-c:values ("GRM" "KGM" "TNE" "STN" "ONZ" "LBR" "CMT" "MTR" "KTM" "INH" "FOT" "YRD" "CMK" "MTK" "INK" "FTK" "YDK" "CMQ" "MTQ" "INQ" "FTQ" "YDQ" "MLT" "LTR" "HLT" "H87" "SET" "PR" "ZP" "KWH" "").
```

Comment:
the empty entry is by intention? If so explain that in the description

Reply:
Was not by intention.

Reply:
Opened PR #299  to reuse model without empty enum value

Reply:
replaced by MS3 model. I'll rebase the branch after merging the model.